doc/SecurityProtocol.md

```diff
@@ -0,0 +1,158 @@
+# Tock Security Vulnerability Handling Process
+
+## Overview
+
+This document outlines the internal process for handling security vulnerabilities in Tock. It establishes clear procedures for managing privately reported security issues, ensuring timely responses, clear communication, and efficient resolution.
+
+## Roles and Responsibilities
+
+### Triage Coordinator
+
+The Triage Coordinator (currently @charles37) is responsible for:
+
+- Initial review of security reports within 48 hours (2 working days)
+- Proper categorization and assignment of vulnerabilities
+- Coordination of the response team
+- communication with stakeholders
+- Tracking progress and ensuring timely resolution
+
+### Security Response Team
+
+The Security Response Team consists of maintainers with expertise in different subsystems who can be called upon to address security vulnerabilities.
+
+## Subsystem Contacts
+
+| Subsystem  | Primary Contact | Secondary Contact |
+| ---------- | --------------- | ----------------- |
+| Kernel     | TBD             | TBD               |
```

Comment:
Based on the previous vulnerabilities that have centered around low-level `arch`-crate related bugs, can we add one row per supported architecture? I'd be volunteering to serve for RISC-V.

Reply:
This is a good idea, and I can be the contact for ARM.